subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -1360,6 +1443,17 @@ static void rp_comm_st_wait_tx_ack(struct ll_conn *conn, struct proc_ctx *ctx, u
 			break;
 		}
 #endif /* CONFIG_BT_CTLR_DATA_LENGTH */
+		case PROC_FRAME_SPACE: {
+			uint8_t fsu_changed = ull_fsu_update_eff(conn);
+
+			fsu_changed |= ctx->data.fsu.ntf_fsu;
```

Comment:
I guess if `fsu_changed` becomes a `bool`, then this needs to be something like 


```suggestion
			uint8_t fsu_changed = ull_fsu_update_eff(conn);

			fsu_changed |= ctx->data.fsu.ntf_fsu != 0U;
```

Reply:
Or may `ntf_fsu` a `bool` too